### DISCUSSION TITLE
BE Viewer on Kali Linux

### DISCUSSION THREAD

**Discussion by smitty92-ay11** [simsong/bulk_extractor#384] [General]
Good Day!

I am a forensics guy transitioning from a windows GUI environment and making the leap to Linux. I have the latest Kali LInux distro installed with BE. I was wondering is there a basic command to launch BE Viewer so that i can have a GUI for BE? thank you.

Josh

**Comment by simsong** [comment]
Yes. You can run the BEViewer jar file and have it talk to the local system. You'll need to look at the source code or ask @BruceMty as I'm not up on it at the moment and I'm dealing with a lot of other fires at the moment..

The BEViewer is being moved here: https://github.com/simsong/BEViewer
However, it is not up and running yet. Do you want to get it up and running? The idea is to have the new repo isolated the GUI stuff and bring in the executables as a sub-project.
